tm-monitor/monitor/monitor.go

```diff
@@ -160,14 +172,17 @@ func (m *Monitor) listen(nodeName string, blockCh <-chan tmtypes.Header, blockLa
 		case b := <-blockCh:
 			m.Network.NewBlock(b)
 			m.Network.NodeIsOnline(nodeName)
+			m.UpdateNodeStatus(nodeName, true)
```

Comment:
it's not clear what `status:bool` is here. let's change it to `NodeIsOnline(nodeName)`

Reply:
do you mean that UpdateNodeStatus related logic can be moved to NodeIsOnline, so the node right status can be got directly in it. But if in this way, how to get the node object? Because Online field in node object will be printed later.

Reply:
no, I meant `renaming UpdateNodeStatus to NodeIsOnline` and removing 2nd argument